packages/wonder-blocks-modal/components/two-column-modal.js

```diff
@@ -78,32 +125,85 @@ const styles = StyleSheet.create({
         top: 8,
     },
 
-    column: {
-        boxSizing: "border-box",
-        flex: "0 0 50%",
+    containerNoSidebar: {
+        minHeight: "auto",
+    },
 
-        // TODO(mdr): Some call sites probably won't want built-in padding. How
-        //     should we handle that possibility?
+    column: {
         paddingTop: 64,
         paddingBottom: 64,
+        minHeight: 464,
+        flex: "1 1 50%",
```

Comment:
I feel like this rule is a bit misleading, because the 50% flex-basis isn't really relevant for the small-sidebar and 1-column cases, but happens to work anyway due to clever subtleties. I'd kinda prefer that we break out the styles for these various cases into entirely different style classes, to help make the expected behavior clearer for future readers.

Reply:
Hmm, ok! I'll see if I can split apart some of these rules to make them clearer (rather than having styles that override each other).